privacy/index.md

```diff
@@ -133,37 +135,12 @@ Our products and other web properties are operated in the United States. If you
 
 ## When transferring personal data from the EU
 
-The GDPR requires that any data transferred out of the EU must be treated with the same level of protection that the EU privacy laws grant. The privacy laws of the United States generally do not meet that requirement. That is why since GDPR went into effect, Basecamp has offered a data processing addendum and voluntarily participated in the EU-US Privacy Shield Framework as well as the Swiss-US Privacy Shield Framework.
+The GDPR requires that any data transferred out of the EU must be treated with the same level of protection that the EU privacy laws grant. The privacy laws of the United States generally do not meet that requirement. Since GDPR went into effect, Basecamp has offered a data processing addendum.
```

Comment:
Would split this out (next time). This repo serves as a public record of our policy proposals & changes, so major material changes should get top billing for customers and followers to see in their notifications.